core/src/main/scala/org/broadinstitute/dsde/rawls/entities/local/LocalEntityProvider.scala

```diff
@@ -35,5 +37,19 @@ class LocalEntityProvider(workspace: Workspace, dataSource: SlickDataSource)
     }
   }
 
-  override def deleteEntities(entRefs: Seq[AttributeEntityReference]): Future[Int] = ???
```

Comment:
I'm kind of amazed `???` works in scala, but neat!

Reply:
you'll get a runtime exception if you try to call it, but it's a nice way to stub things out!